src/api/app/models/comment.rb

```diff
@@ -59,7 +59,7 @@ def notification_parameters(params = {})
 
   private
 
-  def create_notification(params = {})
+  def create_event(params = {})
     params = notification_parameters(params)
```

Comment:
`notification_parameters` not?

Reply:
:thinking:  I'm not sure I understand what is wrong with this...

Reply:
Shouldn't it turn from `create_notification` & `notification_parameters` into `create_event` & `event_parameters`?

Reply:
I've checked and now it is renamed everywhere.